themes/light.json

```diff
@@ -5,164 +5,129 @@
     "foreground": "#444d56",
     "descriptionForeground": "#6a737d",
     "errorForeground": "#cb2431",
-
```

Comment:
The reason these lines got removed is because this file is now auto-generated. I have intentionally not modified any theme config/colors as I wanted this PR to purely focus on auto-generation first.